pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -34,10 +34,25 @@ type Code int
 // NodeScoreList declares a list of nodes and their scores.
 type NodeScoreList []NodeScore
```

Comment:
Should we move `NodeScoreList`, `NodeScore` and other types in this file to `pkg/scheduler/apis/extender/v1`?

Reply:
`pkg/scheduler/apis/extender/v1` is primarily for API backward compatibility, I don't want and shouldn't move Scheduler Framework related stuff there.

Right now, some Framework API fiels are transitively related with Scheduler ComponentConfig, so they're put into `pkg/scheduler/apis/config/v1alpha1`.